src/Form.svelte

```diff
@@ -22,6 +39,22 @@
   export let disabled = false;
   export let config: Config;
 
+  let formFields = fields;
+
+  const check = (e: any) => {
+    formFields = formFields.map(field => {
+      if (field.name === e?.target?.name) {
+        const hasErrors = field.validate && !validationTypes[field.validate].test(e?.target?.value);
+        submitDisabled = hasErrors ? true : false;
```

Comment:
Can we just move `hasErrors` to the outer scope, and replace `submitDisabled` completely?

Reply:
Yeah, we can add the `field.validate` check to the if statement and remove `submitDisabled`.